value.go

```diff
@@ -178,13 +201,15 @@ func (v Value) IsStringy() bool {
 }
 
 func (v Value) IsError() bool {
-	return v.Type == TypeError
+	return AliasOf(v.Type) == TypeError
 }
 
-var missingAsBytes = []byte(missing)
-
 func (v Value) IsMissing() bool {
-	return v.Type == TypeError && bytes.Equal(v.Bytes, missingAsBytes)
+	return v.Type == TypeError && bytes.Equal(v.Bytes, missing)
+}
+
+func (v Value) IsQuiet() bool {
+	return v.Type == TypeError && bytes.Equal(v.Bytes, quiet)
```

Comment:
Maybe use Missing and Quiet here? And should v.Type be AliasOf(v.Type)?
```suggestion
	return v.Type == Missing.Type && bytes.Equal(v.Bytes, Missing.Bytes)
}

func (v Value) IsQuiet() bool {
	return v.Type == Quiet.Type && bytes.Equal(v.Bytes, Quiet.Bytes)
```

Reply:
This is nice because it will still work when we changed to structured errors.  Missing and Quiet are internally created things and I'm not sure we should support customized variations of them with aliases, but maybe there's a use case for that?  We can also ad AliasOf later...

Reply:
Actually this is a little complicated because Missing and Quiet will need a type context when we have structured errors...